modules/about-rhde.adoc

```diff
@@ -21,30 +21,36 @@ For a non-technical description of {product-title}, see link:https://www.redhat.
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/{op-system-version-major}/html/9.2_release_notes/index[{op-system-ostree-first}] 9.2 release notes contain details about {op-system-ostree}
 
-* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/{op-system-version-major}/html/9.3_release_notes/index[{op-system-ostree-first}] 9.3 release notes contain details about {op-system-ostree}
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/{op-system-version-major}/html/9.3_release_notes/index[{op-system-ostree-first}] 9.3 release notes contain details about {op-system-ostree}
 
-* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/{op-system-version-major}/html/9.4_release_notes/index[{op-system-ostree-first}] 9.4 release notes contain details about {op-system-ostree}
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/{op-system-version-major}/html/9.4_release_notes/index[{op-system-ostree-first}] 9.4 release notes contain details about {op-system-ostree}
 
-* link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{ansible-version}/html/red_hat_ansible_automation_platform_release_notes/index[{ansible} Release Notes]
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/{op-system-version-major}/html/9.6_release_notes/index[{op-system-ostree-first}] 9.6 release notes contain details about {op-system-ostree}
+
+* link:https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/{ansible-version}/html/red_hat_ansible_automation_platform_release_notes/index[{ansible} Release Notes]
 
 [id="device-edge-compatibility_{context}"]
 == {product-title} product release compatibility matrix
 
-{op-system-first} and {microshift} work together as a single solution for device-edge computing. You can update each component separately, but the product versions must be compatible. For example, an update of {microshift} from 4.14 to 4.16 requires a {op-system} update. Supported configurations of {product-title} use verified releases for each together as listed in the following table:
+{op-system-first} and {microshift-short} work together as a single solution for device-edge computing. You can update each component separately, but the product versions must be compatible. For example, an update of {microshift-short} from 4.14 to 4.16 requires a {op-system-base} update. Supported configurations of {product-title} use verified releases for each together as listed in the following table:
 
 [%header,cols="3",cols="1,1,2"]
 |===
 ^|*{op-system-base} Version(s)*
 ^|*{microshift-short} Version*
 ^|*Supported {microshift-short} Version{nbsp}&#8594;{nbsp}Version Updates*
 
+^|9.6
+^|4.19
+^|4.19.0{nbsp}&#8594;{nbsp}4.19.z
+
 ^|9.4
 ^|4.18
-^|4.18.0{nbsp}&#8594;{nbsp}4.18.z
+^|4.18.0{nbsp}&#8594;{nbsp}4.18.z, 4.18{nbsp}&#8594;{nbsp}4.19
```

Comment:
```suggestion
^|4.18.0{nbsp}&#8594;{nbsp}4.18.z, 4.18{nbsp}&#8594;{nbsp}4.19  on {op-system-base} 9.6
```
I saw we have  this "... on RHEL ..." in the rows for V4.14 and V4.15, which I think is  good, to make it very explict that this update does require an OS version upgrade. Thats important for customers to understand, as it might otherwise end in an unsupported configuration.